structuralcodes/geometry/_reinforcement.py

```diff
@@ -125,3 +122,127 @@ def add_reinforcement_line(
             group_label=group_label,
         )
     return geo
+
+
+def add_reinforcement_circle(
```

Comment:
This is a good addition. Can we make it even more consistent with the `add_reinforcement_line` function? In the latter function, we have `geo, coords_i, coords_j, ...` as the first arguments in the signature, i.e. after the geometry we have arguments targeting the line to put reinforcement along. The equivalent in `add_reinforcement_circle` could be to have `geo, center, radius, start_angle, stop_angle, ...`. What do you think? But you probably implemented this order to have keyword arguments after positional arguments...

Reply:
Got it. I am fine with that 👍